elastic/integrations
# [Cloud Security] add observer.vendor field mappings

## Proposed commit message

Add `observer.vendor` to csp field mappings

My expectation was that as `ecs@mappings` component template is attached to our latest index templates, the ecs field mappings would appear automatically, but that wasn't the case



## Checklist


## Author's Checklist



## How to test this PR locally



1. build with `elastic-package build`
2. run stack with this package via `elastic-package stack up -v -d`
3. ingest data into findings and vulnerability datastream, check that observer.vendor is provided and the mapping for this field exists on CSP data view and latest indexes. The observer.vendor should be searchable on the Findings page and available in the data grid field browser (possible to add as a column)

## Related issues


- contributes to https://github.com/elastic/kibana/issues/188802
- follow up after https://github.com/elastic/integrations/pull/10945

## Screenshots
